### DISCUSSION TITLE
Compiling libigl with Rad Studio C++ Compiler

### DISCUSSION THREAD

**Discussion by German098** [libigl/libigl#2171] [Q&A]
Hi, I have used visual studio Community 2022 to compile the libigl library, and I had no problem. But now, I want to try to compile the library in Rad Studio C++ with the BCC64 compiler (64 bits and it is based on Clang: [https://docwiki.embarcadero.com/RADStudio/Sydney/en/BCC64](https://docwiki.embarcadero.com/RADStudio/Sydney/en/BCC64)) I just wanted to know if this can be done.

Best regards :).

**Comment by alecjacobson** [comment]
Cool! I have no idea if it will work. Would be happy to here how it goes.
